src/components/apply/formNormalizers.js

```diff
@@ -1,15 +1,18 @@
 import _ from 'lodash';
 
-export const simpleNormalizer = pred => (input, prevInput) => pred(input) || input === '' ? input : prevInput;
+export const simpleNormalizer = pred =>
+  (input, prevInput) => pred(input) || input === '' ? input : prevInput;
```

Comment:
I personally find this ternary a little hard to read. Maybe parens of an if statement?